bundle/store.go

```diff
@@ -959,6 +959,9 @@ func applyPatches(ctx context.Context, store storage.Store, txn storage.Transact
 
 		// apply the patch
 		if err := store.Write(ctx, txn, op, path, pat.Value); err != nil {
+			if op == storage.RemoveOp {
+				return suppressNotFound(err)
```

Comment:
As per the [spec](https://datatracker.ietf.org/doc/html/rfc6902#section-4.2), the target path MUST exist. Maybe it would be a good idea to update the [docs](https://www.openpolicyagent.org/docs/latest/management-bundles/#delta-bundle-patch-operations) to reflect that. We can update the description of the `remove` operation to include the line 
```
The target path must exist for the operation to be successful.
```

Reply:
Oh yeah okk. I reverted the change and updated the doc then.